### PR TITLE
fix bugsnag error

### PR DESCRIPTION
**Pull request by Kamsiy**
Parse error messages are sent to bugsnag when logs are empty. This PR provides a fix to avoid this

### CODE REVIEW

File: lib/sarif/bandit_sarif.rb [hunk 1]
```diff
@@ -11,9 +11,13 @@ def initialize(scan_report)
     end
 
     def parse_scan_report!
-      parsed_result = JSON.parse(@scan_report.log(''))
-
-      parsed_result['results'].concat(parsed_result['errors'])
+      logs = @scan_report.log('')
```
Comment (joshua-ostrom) [inline lib/sarif/bandit_sarif.rb:14]: return [] if logs.size.zero?

File: lib/sarif/brakeman_sarif.rb [hunk 2]
```diff
@@ -11,8 +11,13 @@ def initialize(scan_report)
     end
 
     def parse_scan_report!
-      parsed_result = JSON.parse(@scan_report.log(''))
-      parsed_result['warnings'].concat(parsed_result['errors'])
+      logs = @scan_report.log('')
```
Comment (joshua-ostrom) [inline lib/sarif/brakeman_sarif.rb:14]: return [] if logs.size.zero?

File: lib/sarif/cargo_audit_sarif.rb [hunk 3]
```diff
@@ -11,14 +11,19 @@ def initialize(scan_report)
     end
 
     def parse_scan_report!
-      x = JSON.parse(@scan_report.log(''))
-      vulnerabilities = x['vulnerabilities']['list'] || []
-      unmaintained = x['warnings']['unmaintained'] || []
-      yanked = x['warnings']['yanked'] || []
-      @logs = vulnerabilities.concat(unmaintained, yanked)
+      logs = @scan_report.log('')
+      if !logs.size.zero?
```
Comment (joshua-ostrom) [inline lib/sarif/cargo_audit_sarif.rb:15]: return [] if logs.size.zero?

File: lib/sarif/gosec_sarif.rb [hunk 4]
```diff
@@ -13,18 +13,23 @@ def initialize(scan_report)
...
     def parse_scan_report!(scan_report)
-      json_obj = JSON.parse(scan_report.log(''))
-      issues = json_obj['Issues']
-      errors = json_obj['Golang errors']
-      parsed_errors = []
-      errors.each do |key|
-        key[1].each do |location|
-          location['uri'] = key[0]
-          parsed_errors << parse_error(location)
+      log = scan_report.log("")
```
Comment (joshua-ostrom) [inline lib/sarif/gosec_sarif.rb:16]: return [] if logs.size.zero?

**Review by joshua-ostrom — CHANGES_REQUESTED** [review]
Changes looks good.  Can we add a spec coverage on this?  (Happy to pair if needed).  I would not let that hold up a patch though.

**Review by ghbren — APPROVED** [review]

**Review by ghbren — APPROVED** [review]

**Review by joshua-ostrom — APPROVED** [review]

**Pull request merged**